src/geolib.js

```diff
@@ -802,7 +802,7 @@
         orderByDistance: function(latlng, coords) {
             const coordsArray = Object.keys(coords).map(function(idx) {
                 const distance = this.getDistance(latlng, coords[idx]);
-                const augmentedCoord = Object.create(coords[idx]);
+                const augmentedCoord = Object.assign({}, coords[idx]);
```

Comment:
```suggestion
                return {
                  ...coords[idx],
                  distance,
                  key: idx
                };
```

I don't intend to hijack your PR, I thought I will suggest an alternative approach for using object.assign(), I am confident this should still solve your problem, I maybe wrong 😄

Reply:
Absolutely. The problem with this is, there are few platforms still using older versions of Nodejs which doesn't support new features like spread operator. For eg. Cloud functions for firebase uses Nodejs v6.x which has no support for spread operator. Object.assign() works anyway :)